lib/mkmf.rb

```diff
@@ -2148,6 +2148,7 @@ def configuration(srcdir)
 DLDFLAGS = $(ldflags) $(dldflags) $(ARCH_FLAG)
 LDSHARED = #{CONFIG['LDSHARED']}
 LDSHAREDXX = #{config_string('LDSHAREDXX') || '$(LDSHARED)'}
+POSTLINK = #{RbConfig.expand('$(POSTLINK)', CONFIG)}
```

Comment:
You should be able to use `RbConfig::CONFIG` instead of `CONFIG` (= `RbConfig::MAKEFILE_CONFIG`).

```suggestion
POSTLINK = #{config_string('POSTLINK', RbConfig::CONFIG)}
```